File: go lib/hurufvokal.c
package main

import (
	"fmt"
	"strings"
)

func main() {
	fmt.Println("##  Program Go Menghitung Jumlah Huruf Vokal  ##")
	fmt.Println("=================================================")

	var x string
	vokal := 0

	fmt.Print("Input kata / kalimat: ")
	fmt.Scanln(&x)
	fmt.Println()

	// Hitung jumlah huruf vokal
	for _, huruf := range strings.ToLower(x) {
		if huruf == 'a' || huruf == 'i' || huruf == 'u' || huruf == 'e' || huruf == 'o' {
			vokal++
		}
	}

	// Tampilkan total huruf vokal jika ditemukan
	if vokal > 0 {
		fmt.Printf("Jumlah huruf vokal = %d\n", vokal)
	} else {
		fmt.Println("Huruf vokal tidak ditemukan")
	}
}

File: go lib/hurufvokal.go
package main  
  
import (  
 "fmt"  
)  
  

func main() {
var kata string

fmt.Print("Masukkan kata: ")
fmt.Scan(&kata)

jumlahVokal := 0

for i:=0; i<len(kata); i++ {
	if kata[i] == 'a' || kata[i] == 'i' || kata[i] == 'u' || kata[i] == 'e' || kata[i] == 'o' || 
	   kata[i] == 'A' || kata[i] == 'I' || kata[i] == 'U' || kata[i] == 'E' || kata[i] == 'O' {
		jumlahVokal++
	}else if kata[i] == " " {
		fmt.Scanln(&kata)
	}
}

fmt.Println("Jumlah huruf vokal dalam kata", kata, "adalah", jumlahVokal)
}
